assignment/scripts/loops-practice.js

```diff
@@ -13,18 +13,27 @@ for (let i=0; i<4; i++) {
 // 1.a. TODO: Write a for loop to console.log the numbers from 0 to 5 
 //   - Which part of the example loop do you need to change to do this?
 console.log('count from 0 to 5');
-
+for (let i = 0; i < 6; i++) {
+  console.log(i);
+}
 // 1.b. TODO: Write a for loop to console.log the numbers from 3 to 5
 //   - Which part of the example loop do you need to change to do this?
 console.log('count from 3 to 5');
-
+for (let i = 3; i < 6; i++) {
+  console.log(i);
+}
 // 1.c. TODO: Write a for loop to console.log EVEN numbers from 2 to 10
 //   - Which part of the example loop do you need to change to do this?
 console.log('count even numbers from 2 to 10 (2, 4, 6, 8, 10):');
+for (let i = 2; i < 11; (i += 2)) {
```

Comment:
The extra parentheses here are a bit non-standard, but if it looks more readable to your eye, who am I to argue?